docs/cable_tree.md

```diff
@@ -0,0 +1,28 @@
+<!--
+ ~ SPDX-FileCopyrightText: 2022 Copyright DB InfraGO AG and the capellambse-context-diagrams contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Cable Tree View Diagram
+
+The `Cable Tree View` diagram visualizes the cable connections between ports. You can access `.cable_tree` on any `pa.PhysicalLink` element.
+
```

Comment:
Maybe we should also inform the user about the port labels being rendered always, and about the effects of not having port labels in the model.

Reply:
I made the port labels optional. No need to inform the user now!